Sandbox/src/Game/Player/Player.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Game/Player/Player.h"
+
+Player::Player()
+{
+	mSprite = nullptr;
+	//mSprite->Parent(this);
+
+	mEatDelay = 0.0f;
+	mDrinkDelay = 0.0f;
+
+	mInCombat = false;
+	mCombatDelay = 0.0f;
+}
+
+Player::~Player()
+{
+	delete mSprite;
+
+}
+
+void Player::SetEatDelay()
+{
+	if (mInCombat)
+	{
+		mCombatDelay += 1.8f - mDrinkDelay;
```

Comment:
confused why setting eat delay uses mDrinkDelay? and same with setDrinkDelay requiring mEatDelay